modules/virt-booting-efi-mode.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-booting-vms-efi-mode_{context}"]
+= Booting virtual machines in EFI mode
+
+Booting a virtual machine in EFI mode allows for a quicker boot process.
```

Comment:
s/quicker/faster

(consider including a comparison to a slower boot mode; what is it faster than?)